pkg/kubelet/cm/container_manager.go

```diff
@@ -118,6 +118,12 @@ type ContainerManager interface {
 
 	// UpdateAllocatedDevices frees any Devices that are bound to terminated pods.
 	UpdateAllocatedDevices()
+
+	// GetAllDevices returns information about all the devices known to the manager
+	GetAllDevices() podresourcesapi.ResourceDeviceMap
```

Comment:
should we call this `GetAllocatableDevices` ?

Reply:
sure, same original rationale as per `GetAllCPUs` above. Changing.